src/apps/backend/modules/vendor_account/types.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass
+from enum import Enum
+
+from modules.account.types import Account
+
+
+class VendorType(Enum):
+    AMAZON = "AMAZON"
+
+
+@dataclass(frozen=True)
+class VendorAccount:
+    account: str | Account
```

Comment:
This is not right.

It should be `account_id`: str and `account: Account` if we must include full `account` object.

Why do we need it through?